src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static FileReaderService fileReaderService;
+    private List<String> fruitList;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Before
+    public void setUp() {
+        fruitList = new ArrayList<>();
+        fruitList.add("b,apple,20");
+        fruitList.add("s,banana,30");
+        fruitList.add("r,apple,10");
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void nullValue_NotOk() {
+        fileReaderService.readFromFile(null);
+    }
+
+    @Test
+    public void readFromFile_Ok() {
+        String pathName = "src\\test\\resources\\testFile.csv";
+        String text = "b,apple,20" + System.lineSeparator() + "s,banana,30"
+                + System.lineSeparator() + "r,apple,10";
+        File file = new File(pathName);
+        try {
+            Files.write(file.toPath(),text.getBytes(StandardCharsets.UTF_8));
```

Comment:
I would recommend not using Files.write here, but simply write to the file by hand